service/src/test/java/uk/gov/hmcts/reform/fpl/service/RespondentServiceTest.java

```diff
@@ -260,4 +275,250 @@ private RespondentSolicitor buildRespondentSolicitor(String email) {
             .build();
     }
 
+    @Nested
+    class RepresentationChanges {
+
+        @Test
+        void shouldReturnEmptyListOfChangesWhenRespondentsNotPresent() {
+            List<ChangeOrganisationRequest> changes = service.getRepresentationChanges(null, null);
+
+            assertThat(changes).isEmpty();
+        }
+
+        @Test
+        void shouldReturnEmptyListOfChangesWhenRespondentsAreEmpty() {
```

Comment:
could be combined with the previous to use a null and empty source parameterised test if you wanted